hedera-node/test-clients/src/main/java/module-info.java

```diff
@@ -102,7 +102,7 @@
     requires com.swirlds.platform.core.test.fixtures;
     requires com.fasterxml.jackson.core;
     requires com.fasterxml.jackson.databind;
-    requires com.github.docker.java.api;
+    requires com.github.dockerjava.api;
```

Comment:
This name is changed, because `com.github.docker.java.api` was our own creation. In the new version, `com.github.dockerjava.api` is the official name.